hash.c

```diff
@@ -3754,11 +3754,11 @@ inspect_hash(VALUE hash, VALUE dummy, int recur)
 
 /*
  *  call-seq:
- *    hash.to_s -> new_string
+ *    hash.inspect -> new_string
  *
  *  Returns a new \String containing the hash entries:
  *    h = {foo: 0, bar: 1, baz: 2}
- *    h.to_s # => "{:foo=>0, :bar=>1, :baz=>2}"
+ *    h.inspect # => "{:foo=>0, :bar=>1, :baz=>2}"
  *
  *  Hash#to_s is an alias for Hash#inspect.
```

Comment:
Do you think it's worth mentioning that it recursively calls inspect? Or is that implicit/obvious?

Reply:
Neither implicit nor obvious, in my view. But also not part of my PR.  (I'm not trying to make the documentation perfect; just better.)